libs/execution_context/ebpf_program.c

```diff
@@ -1559,6 +1559,13 @@ ebpf_program_invoke(
             ebpf_program_entry_point_t function_pointer;
             function_pointer = (ebpf_program_entry_point_t)(current_program->code_or_vm.code.code_pointer);
             *result = (function_pointer)(context);
+#ifdef _DEBUG
```

Comment:
There are currently no ifdef _DEBUG lines anywhere in this directory.  Other log calls check the state inside the macro, why are ifdefs needed here?

Reply:
Thanks. The intent was to add the trace only in DEBUG image with INFO level. Did you mean to change the log level to LEVEL_VERBOSE, and remove #ifdef?

Reply:
yes that would be better